app/views/users/show.html.erb

```diff
@@ -1,3 +1,13 @@
+<div class="show-page">
+<div class="users">
+<%= render 'shared/user', user:@user %>
+</div>
+<div class="posts">
+<h2>Recent Posts</h2>
+<% @recent_posts.each do |post| %>
+<%= render 'shared/post', post:post, user:@user %>
+<% end %>
+</div>
 <%= stylesheet_link_tag('index') %>
 <div class="container">
   <div class="user">
```

Comment:
- [ ] You have a css file you are trying to use that you did not create in the CSS assets pipeline/folder, 
![image](https://github.com/firstpj/blog-app/assets/86200356/35b84cf2-19b0-4492-89da-eb0820e3b3f1) 
### The error I encountered
![image](https://github.com/firstpj/blog-app/assets/86200356/f5870214-a079-46e8-aa74-6307978a7080)
when I created the file, I was able to pass this issue to the next one, kindly fix this, and let us move to the next issue.